### PR TITLE
Update to_duration signature

### PR DESCRIPTION
**Pull request by edvorg**
update to_duration to accept anything that is convertible into Option<Duration>

### CODE REVIEW

**Comment by edvorg** [comment]
@ItsEthra could you please have a look?

**Comment by ItsEthra** [comment]
Maybe a good idea would be to have the `set_duration` function accept `impl Into<Option<Duration>>` instead?

**Comment by edvorg** [comment]
@ItsEthra sounds great! Updated the branch with suggested changes

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ let mut toasts = Toasts::default();
 
 ```rust
 // somewhere within [egui::App::update]...
-toasts.info("Hello world!").set_duration(Duration::from_secs(5));
+toasts.info("Hello world!").set_duration(Some(Duration::from_secs(5)));
```
Comment (ItsEthra) [inline README.md:21]: I think you forgot to remove that

**Review by ItsEthra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (edvorg) [reply README.md]: Right, it's no longer need, updated

**Comment by ItsEthra** [comment]
Thanks.

**Pull request merged**